README.md

```diff
@@ -38,9 +30,17 @@ Bytes, Distributed.
 </div>
 <br/>
 
+## Features
```

Comment:
Should we expand this with the full set of items instead of just the 3 layers? What should we include here?

Reply:
probably, but lets get the existing improvements merged now, and do that in a follow up